### PR TITLE
Upgrade `SDL_ttf `to `2.22.0`, add test for a bugfix

### PR DESCRIPTION
**Pull request by Matiiss**

### CODE REVIEW

File: test/font_test.py [hunk 1]
```diff
@@ -318,6 +318,23 @@ def test_render_multiline(self):
...
+            return
+
+        f = pygame_font.Font(None, 20)
+        one_newline = f.render("\n", True, "black", "white")
+        two_newlines = f.render("\n\n", True, "black", "white")
+        three_newlines = f.render("\n\n\n", True, "black", "white")
+        self.assertTrue(
+            one_newline.get_height()
+            < two_newlines.get_height()
+            < three_newlines.get_height()
```
Comment (ankith26) [inline test/font_test.py:335]: could we improve this test, maybe test that the height of `two_newlines` is roughly 2x the height of `one_newline` and stuff?

Comment (Matiiss) [reply test/font_test.py:335]: Alright, I added two more test cases that check exactly that, but really the main thing this test is for is to make sure `font.render` doesn't raise an exception when supplied two or more newlines.

**Review by ankith26 — APPROVED** [review]
LGTM, thanks for the PR :tada:

**Review by Starbuck5 — APPROVED** [review]
Thanks!

**Pull request merged**